apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -505,14 +505,16 @@ def render(element_html: str, data: pl.QuestionData) -> str:
             )
 
         if grading_method is GradingMethodType.UNORDERED:
-            help_text += "<br>Your answer ordering does not matter. "
+            help_text += "<p>Your answer ordering does not matter. </p>"
         elif grading_method is not GradingMethodType.EXTERNAL:
-            help_text += "<br>The ordering of your answer matters and is graded."
+            help_text += "<p>The ordering of your answer matters and is graded.</p>"
         else:
-            help_text += "<br>Your answer will be autograded; be sure to indent and order your answer properly."
+            help_text += "<p>Your answer will be autograded; be sure to indent and order your answer properly.</p>"
 
         if check_indentation:
-            help_text += "<br><b>Your answer should be indented. </b> Indent your tiles by dragging them horizontally in the answer area."
+            help_text += "<p><b>Your answer should be indented. </b> Indent your tiles by dragging them horizontally in the answer area.</p>"
```

Comment:
Nit (see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b#usage_notes):

```suggestion
            help_text += "<p><strong>Your answer should be indented.</strong> Indent your tiles by dragging them horizontally in the answer area.</p>"
```